staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -392,6 +392,10 @@ loop:
 				continue
 			}
 			newResourceVersion := meta.GetResourceVersion()
+			// SWAT Event lost: print out resourceVersion, EventType and event object name
+			// if event is for a pod or a node
+			fmt.Printf("SWAT,reflector/watchHandler,%s,%s,%s\n",
```

Comment:
I see reflector being used in multiple components (both in server and clients). Can we add `reflector.name` in the log to identify which component we are looking at?